src/style/buttons.less

```diff
@@ -14,6 +14,13 @@
 	.btn-lg {
 		border-radius: 12px;
 	}
+	/* switch off rounding for input appends; copy paste from BS 'cos we changed border-radius above */
```

Comment:
does BS mean 'bootstrap'?

Reply:
Yes (I write Bootstrap so often in comments that abbreviating it feels useful)